frontend/src/pages/JobApplications.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+	<div class="">
+		<header
+			class="sticky top-0 z-10 flex items-center justify-between border-b bg-surface-white px-3 py-2.5 sm:px-5"
+		>
+			<Breadcrumbs
+				class="h-7"
+				:items="[
+					{ label: __('Jobs'), route: { name: 'Jobs' } },
+					{
+						label: applications.data?.[0]?.job_title,
+						route: { name: 'JobDetail', params: { job: props.job } },
+					},
+					{ label: __('Applications') },
+				]"
+			/>
+		</header>
+		<div class="max-w-4xl mx-auto pt-5 p-4">
+			<div class="mb-6">
+				<h1 class="text-lg font-semibold text-ink-gray-9 mb-2">
+					{{ applications.data?.length || 0 }}
+					{{
+						applications.data?.length === 1
+							? __('Application')
+							: __('Applications')
+					}}
+				</h1>
+			</div>
+
+			<ListView
+				v-if="applications.data?.length"
+				:columns="applicationColumns"
+				:rows="applicantRows"
+				row-key="name"
+				:options="{
+					showTooltip: false,
+					selectable: false,
+				}"
+			>
+				<ListHeader
+					class="mb-2 grid items-center space-x-4 rounded bg-surface-gray-2 p-2"
+				>
+					<ListHeaderItem :item="item" v-for="item in applicationColumns" />
+				</ListHeader>
+				<ListRows>
+					<ListRow
+						:row="row"
+						v-slot="{ column, item }"
+						v-for="row in applicantRows"
+						class="cursor-pointer"
+					>
+						<ListRowItem :item="item">
+							<div
+								v-if="column.key === 'display_name'"
+								class="flex items-center space-x-3"
+							>
+								<img
+									v-if="row.user_image"
+									:src="row.user_image"
+									:alt="row.full_name"
+									class="w-8 h-8 rounded-full object-cover"
+								/>
+								<div
+									v-else
+									class="w-8 h-8 rounded-full bg-surface-gray-3 flex items-center justify-center"
+								>
+									<User class="w-4 h-4 text-ink-gray-6" />
+								</div>
+								<span class="text-sm font-medium">{{ item }}</span>
+							</div>
+							<div
+								v-else-if="column.key === 'actions'"
+								class="flex justify-center space-x-2"
+							>
+								<Button
+									v-if="row.resume"
```

Comment:
Lets just show three dots here, that opens a menu with options to "View Resume" or "Send Email"